src/w2grid.js

```diff
@@ -7098,7 +7101,7 @@ class w2grid extends w2base {
                     let gText = w2utils.lang(typeof colg.text == 'function' ? colg.text(colg) : colg.text)
                     tmpf      = '<td id="grid_'+ obj.name + '_column_' + ii +'" class="w2ui-head" col="'+ ii + '" '+
                            '        colspan="'+ colspan +'">'+
-                           '    <div class="w2ui-col-group">'+
+                           '    <div class="w2ui-col-group"' + colg.applystyle + '>'+
```

Comment:
I think it would be better to do use `colg.style` here directly, rather then assigning it to a variable above
`... style="' + colg.style + '">...`